minifi-bootstrap/pom.xml

```diff
@@ -43,6 +43,10 @@ limitations under the License.
         </dependency>
         <dependency>
             <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-properties</artifactId>
```

Comment:
Instead of adding another pom dependency for one static helper method I would prefer to just create a new private one that does it:
    private static boolean nullOrEmpty(String input) {
        return input == null || input.isEmpty();
    }